lib/skywalking/util.lua

```diff
@@ -119,4 +119,50 @@ else
 end
 
 
+if _M.is_ngx_lua then
```

Comment:
Could we provide a fall back to no-pool mechanism? But pool could be default. If there is a bug of this implementation, we have a chance to disable this.

Reply:
added, please take a look when you have time